packages/integration/src/bitbucket/config.ts

```diff
@@ -86,14 +96,35 @@ export function readBitbucketIntegrationConfig(
     );
   }
 
+  if (baseUrl) {
+    baseUrl = trimEnd(baseUrl, '/');
+  } else {
+    baseUrl = `https://${host}`;
```

Comment:
Along with the above, one thing to consider is whether we should bother inferring the value at all. 
If we infer it, then it will always be available in the config. Which could have downstream effects. 

For example, test cases had to be updated because they were missing the property in their mock objects.